src/common/paths/buckets_bucketID_owners.yml

```diff
@@ -29,31 +72,96 @@ post:
   tags:
     - Buckets
   summary: Add an owner to a bucket
+  description: |
+    Adds an owner to a bucket and returns the [owners]({{% INFLUXDB_DOCS_URL %}}/reference/glossary/#owner)
+    with role and user detail.
+
+    Assigns a bucket `owner` role to a user and returns the new owner detail.
```

Comment:
Can we combine these sentences?

Reply:
I missed this and actually meant to remove the second sentence as it has a duplicate meaning. Thanks for catching this